services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusService.kt

```diff
@@ -126,7 +126,7 @@ class HakemusService(
         }
 
         assertGeometryValidity(request.areas) { validationError ->
-            "Invalid geometry received when updating application id=${applicationEntity.id}, reason=${validationError.reason}, location=${validationError.location}"
+            "Invalid geometry received when updating application ${applicationEntity.logString()}, reason=${validationError.reason}, location=${validationError.location}"
```

Comment:
There are still some "application" among "hakemus" strings in log messages.

Reply:
I changed every log message in HakemusService to hakemus instead of application.